docs/release/trg-0/trg-1-08.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TRG 1.08 - Interfaces Documentation
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History          |
+|------------|--------------|-----------------------|
+| Draft      | 31-Jan-2024  | created initial Draft |
+
+## Why
+
+Interfaces Documentation is crucial for integration testing and ensuring seamless interoperability for Developers consuming the product API
```

Comment:
i think it is not only crucial for testing, but in general for everyone aiming to consume the API.
Not sure if the API docs _ensure_ interoperability